package.json

```diff
@@ -4,34 +4,38 @@
   "description": "",
   "main": "index.ts",
   "scripts": {
-    "build": "npx tsc",
-    "start": "npx tsc && node dist/index.js",
-    "dev": "concurrently \"npx tsc --watch\" \"nodemon -q dist/index.js\"",
+    "build": "npx etsc",
+    "start": "npx etsc && node dist/index.js",
+    "dev": "concurrently \"npx etsc --watch\" \"nodemon -q dist/index.js\"",
     "swagger": "node swaggerGen.ts",
```

Comment:
A `postbuild` script might be helpful here, that runs the `swagger` script automatically after every build! Something like this:

```json
"scripts": {
  ...
  "postbuild": "npm run swagger"
  ...
}
```

Reply:
The swagger isn't functional right now.

Reply:
but yes, good pointer.

Reply:
Refer to this: https://github.com/leveorxyz/express-boilerplate-ts/issues/5